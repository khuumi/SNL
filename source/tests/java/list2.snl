start list:
	list is [1, "a", true, 1.0]
	do show to list
done