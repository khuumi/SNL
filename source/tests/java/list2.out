[ 1, a, true, 1.0 ]
